public/src/components/Contact/Contact.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
```

Comment:
Pages that are routed should be in their own directory not in the components directory

Reply:
This Component should really be at the location 
`public/src/contact/Contact.js`

Reply:
Im not understanding this. Are they not components? Also, the Shop component was built with a seperate directory that pulls in the shop but the actual shop component still sits in the components directory? Why is that?

Reply:
Yeah we should probably rename the folder `components` to `shared-components` so that it makes more sense

Reply:
If you never organize the components some how it will get way too large at some point